src/arrayMethodSort.js

```diff
@@ -5,8 +5,26 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    for (let i = 0; i < this.length - 1; i++) {
+      let minIndex = i;
+
+      for (let j = i + 1; j < this.length; j++) {
```

Comment:
Avoid using nested loops. It's bad for performance and actually makes no sense here. 
`this[i]` - current element
`this[i + 1]` - next element